BUILD.bazel

```diff
@@ -615,6 +616,37 @@ internal_php_proto_library(
         "//php:__subpackages__",
     ],
 )
+internal_copy_files(
```

Comment:
I think this might be unnecessary now.  In the ruby/python case we also needed to copy the WKT, which wasn't required here

Reply:
I cleaned up the bazel files, and I believe it's much cleaner / better than before: e039fa0335bfdb4b4c91d1d9282fb7b5715b6cca